Update system-requirements.de.md

Mynyx opened the pull request:
Soweit ich das durch den Vergleich der Readme (https://github.com/contao/managed-edition/blob/4.8/README.md und https://github.com/contao/managed-edition/blob/4.9/README.md) feststellen konnte, hat sich außer der PHP-Version an den Systemvoraussetzungen nichts geändert.

aschempp commented:
Thanks for the contribution!

For the german version, we also always used "true/false" because that would be the correct PHP value. Could you update that?
Also maybe add a description that this is what the backend member view looks like?

Your first pull request was for Contao 3.2, and now for 3.4. If I correctly remember, the changes are already available in 3.2 right? I will then correctly merge everything.

Pretty sure @lionel-m will provide a french translation :-)

bezin commented:
thanks for your comments. I added a reference to the backend member list in both the german and english versions. And yes, you are right about the availability in Contao 3.2 (https://github.com/contao/core/blob/support/3.2/system/modules/core/dca/tl_member.php#L58).

aschempp commented:
Cherry-picked into 3.2+. @lionel-m might want to add a french translation.

Mynyx committed:
Update system-requirements.de.md

Soweit ich das durch den Vergleich der Readme (https://github.com/contao/managed-edition/blob/4.8/README.md und https://github.com/contao/managed-edition/blob/4.9/README.md) feststellen konnte, hat sich außer der PHP-Version an den Systemvoraussetzungen nichts geändert.
